packages/clients/src/aptos/wallet.ts

```diff
@@ -205,7 +212,7 @@ export class WalletClient extends ShinamiRpcClient {
 
   /**
    * Executes a MultiAgentTransaction gaslessly with the specified wallet as the sender.
-   * To call this method, your access key must be authorized for both the Aptos Wallet Service and Aptos Gas Station.
+   * To call this method, your access key must be authorized for both the Wallet Service and Gas Station.
```

Comment:
And, in the case of Aptos, Node Service: https://app.shortcut.com/shinami/story/5347/update-aptos-wallet-service-to-use-our-customer-s-node-service-key-rights-and-not-ours